### PR TITLE
improvement: OrderForm: added helping messages for checkboxes

### PR DESCRIPTION
**Pull request by dmytroshch**
https://github.com/bitfinexcom/ufx-ui/pull/82

ticket - https://app.asana.com/0/1125859137800433/1200564160940294

Now if user hovers cursor to checkbox's placeholder in atomic orders form, it is showing helping information about current option, as it works on web
Снимок экрана от 2021-07-01 14-54-46

Снимок экрана от 2021-07-01 15-03-53

### CODE REVIEW

File: src/util/ui.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ import _reverse from 'lodash/reverse'
 // takes a number as input and returns a localised version with semicolons in it
 // e.g. '123456789.445566' -> '123,456,789.445566'
 export const localiseNumber = (x) => {
-  return _toString(x).replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',')
+  const rez = _toString(x).replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',')
```
Comment (Blaumaus) [inline src/util/ui.js:12]: why do we need these changes?

**Review by Blaumaus — CHANGES_REQUESTED** [review]
Good job with a minor proposal

File: src/util/ui.js (hunk 1 above)
Comment (dmytroshch) [reply src/util/ui.js:12]: accidentally

**Review by bhoomij — APPROVED** [review]
looks good, please keep in DRAFT until relevant change in ufx-ui is published

**Review by bhoomij — APPROVED** [review]

**Review by bhoomij — CHANGES_REQUESTED** [review]
@dmytroshch I just noticed that we now have help cursor even for Checkbox that do not have tooltip-help (attached screencast, could you please fix it ? )


https://user-images.githubusercontent.com/29878604/126151864-fe9dc6ef-50b0-4b6b-b863-45c6b0fb93b0.mov

I think we should revert this change https://github.com/bitfinexcom/ufx-ui/pull/82/files#diff-ebc8b0fa9a1693f8d1722e21806e5c6cc59f919061d6cb056d7fc1bb6f782edcL61

**Review by bhoomij — APPROVED** [review]

**Pull request merged**